core/genesis.go

```diff
@@ -238,13 +238,6 @@ func SetupGenesisBlock(db ethdb.Database, genesis *Genesis) (*params.ChainConfig
 	if height == nil {
 		return newcfg, stored, fmt.Errorf("missing block number for head header hash")
 	}
-
-	// Quorum
-	if storedcfg.PrivacyEnhancementsBlock == nil {
```

Comment:
Why has this been removed?

Reply:
not removed in the latest commit. I think that was an intermediary commit when I pulled the changes that were done in another PR (not merged to `master`).